meson_options.txt

```diff
@@ -0,0 +1 @@
+option('skip_manpages', type: 'boolean', value: false, description: 'Do not build manpages')
```

Comment:
Usually, positive logic is easier to follow, so what about:

```
option('build_manpages', type: 'boolean', value: true, description: 'Build manpages')
```

And then in `meson.build`:

```
if get_option('build_manpages')
    ...
endif
```